input/pagecontent/sushi.md

```diff
@@ -241,13 +249,13 @@ This will run the HL7 IG Publisher, which will take several minutes to complete.
 
 > **Note:** `_genonce` embeds the command `JAVA -jar input-cache/org.hl7.fhir.publisher.jar -ig ig.ini`. If the publisher jar or `ig.ini` file are different locations, the command can be adjusted accordingly.
 
-### IG Publisher Integration (Autobuild Configuration)
+### New! IG Publisher Integration (Autobuild Configuration)
 
 The IG Publisher version 1.0.75 and higher includes native support for FHIR Shorthand and SUSHI. The IG Publisher launches SUSHI and runs it if it detects a folder named **/fsh**. Not having to run SUSHI separately is a minor benefit, but there is a significant advantage related to the _autobuild_ process.
 
-Autobuild is an action that is triggered when you commit IG sources to a Github repository hosted on https://github.com/HL7. Autobuild starts the IG Publisher, which does error checking and, if successful, publishes your IG to http://build.fhir.org. With SUSHI integration, you check in your FSH files to your github repository on https://github.com/HL7, and SUSHI and the IG Publisher will run automatically. 
+[Autobuild](https://github.com/FHIR/auto-ig-builder/blob/master/README.md) is a build service triggered when you commit IG source code to a Github repository. Autobuild can be configured to work from any Github repository, but is pre-configured to run from repositories hosted on https://github.com/HL7. Autobuild starts the IG Publisher, which does error checking and, if successful, publishes your IG to http://build.fhir.org. With SUSHI integration, you check in your FSH files to your github repository on https://github.com/HL7, and SUSHI and the IG Publisher will run automatically.
```

Comment:
> but is pre-configured to run from repositories hosted on https://github.com/HL7

I don't know what you mean by this.  I'm not aware of any preconfiguration on HL7 projects.  You still have to setup the webhook yourself on even those projects.